compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -6,6 +6,7 @@
 #include "llvm/IR/GlobalVariable.h"
 #include "llvm/IR/Instructions.h"
 #include "llvm/IR/Intrinsics.h"
+#include "llvm/IR/Mangler.h"
```

Comment:
Nice catch, but curious why it didn't fail before.

Reply:
This happens regularly when relying on indirect header dependencies. Probably a header include was replaced with a forward declaration somewhere, so we need to explicitly include it now.